src/main/java/com/minecolonies/commands/ColonyInfo.java

```diff
@@ -0,0 +1,144 @@
+package com.minecolonies.commands;
+
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.ColonyManager;
+import com.minecolonies.colony.IColony;
+import com.mojang.authlib.GameProfile;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.TextComponentString;
+import org.jetbrains.annotations.NotNull;
+import javax.annotation.Nullable;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+import static java.util.Collections.emptyList;
+
+/**
+ * List all colonies.
+ */
+public class ColonyInfo extends AbstractSingleCommand
+{
+
+    private static final String ID_TEXT                    = "§2ID: §f";
+    private static final String NAME_TEXT                  = "§2 Name: §f";
+    private static final String MAYOR_TEXT                 = "§2Mayor: §f";
+    private static final String COORDINATES_TEXT           = "§2Coordinates: §f";
+    private static final String COORDINATES_XYZ            = "§4x=§f%s §4y=§f%s §4z=§f%s";
+    private static final String CITIZENS                   = "§2Citizens: §f";
+    private static final String NO_COLONY_FOUND_MESSAGE    = "Colony with mayor %s not found.";
+    private static final String NO_COLONY_FOUND_MESSAGE_ID = "Colony with ID %d not found.";
+
+    public static final String DESC                        = "info";
+
+    /**
+     * Initialize this SubCommand with it's parents.
+     *
+     * @param parents an array of all the parents.
+     */
+    public ColonyInfo(@NotNull final String... parents)
+    {
+        super(parents);
+    }
+
+    @NotNull
+    @Override
+    public String getCommandUsage(@NotNull final ICommandSender sender)
+    {
+        return super.getCommandUsage(sender) + "<ColonyId|OwnerName>";
+    }
+
+    @Override
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
+    {
+        int colonyId = -1;
+        UUID mayorID = sender.getCommandSenderEntity().getUniqueID();
+
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "found" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1481)